examples/app_multi_node/app_pl_work.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import torch
+
+import lightning.app as L
```

Comment:
I suggest we then modify all examples to `lightning.app` in a separate PR, since they will likely copy-pasted and ran from environments where the `lightning` package is installed.

Reply:
```suggestion
import lightning as L
```